tests/unit_tests/udf_wrapper_params/token_classification/error_not_cached_multiple_model_multiple_batch.py

```diff
@@ -1,113 +1,54 @@
 from pathlib import PurePosixPath
 from exasol_udf_mock_python.connection import Connection
-from tests.unit_tests.udf_wrapper_params.token_classification.\
-    mock_token_classification import \
-    MockTokenClassificationFactory, MockTokenClassificationModel, MockPipeline
+from tests.unit_tests.udf_wrapper_params.token_classification.make_data_row_functions import make_input_row, \
+    make_output_row, make_input_row_with_span, make_output_row_with_span, bucketfs_conn, \
+    sub_dir, text_docid, text_start, text_end, agg_strategy_simple, make_model_output_for_one_input_row, make_number_of_strings, text_data
 
 
-def udf_wrapper():
-    from exasol_udf_mock_python.udf_context import UDFContext
-    from exasol_transformers_extension.udfs.models.token_classification_udf import \
-        TokenClassificationUDF
-    from tests.unit_tests.udf_wrapper_params.token_classification. \
-        mock_sequence_tokenizer import MockSequenceTokenizer
-    from tests.unit_tests.udf_wrapper_params.token_classification.\
-        error_not_cached_multiple_model_multiple_batch import \
-        ErrorNotCachedMultipleModelMultipleBatch as params
-
-    udf = TokenClassificationUDF(
-        exa,
-        batch_size=params.batch_size,
-        pipeline=params.mock_pipeline,
-        base_model=params.mock_factory,
-        tokenizer=MockSequenceTokenizer)
-
-    def run(ctx: UDFContext):
-        udf.run(ctx)
-
-def work_with_span_udf_wrapper():
-    from exasol_udf_mock_python.udf_context import UDFContext
-    from exasol_transformers_extension.udfs.models.token_classification_udf import \
-        TokenClassificationUDF
-    from tests.unit_tests.udf_wrapper_params.token_classification. \
-        mock_sequence_tokenizer import MockSequenceTokenizer
-    from tests.unit_tests.udf_wrapper_params.token_classification.\
-        error_not_cached_multiple_model_multiple_batch import \
-        ErrorNotCachedMultipleModelMultipleBatch as params
-
-    udf = TokenClassificationUDF(
-        exa,
-        work_with_spans=True,
-        batch_size=params.batch_size,
-        pipeline=params.mock_pipeline,
-        base_model=params.mock_factory,
-        tokenizer=MockSequenceTokenizer)
-
-    def run(ctx: UDFContext):
-        udf.run(ctx)
-
 class ErrorNotCachedMultipleModelMultipleBatch:
     """
     not cached error, multiple model, multiple batch
     """
-    expected_model_counter = 0
+    expected_model_counter = 1
```

Comment:
Same as https://github.com/exasol/transformers-extension/pull/270/files#r1845243573 I guess

Reply:
Improve docstring

Reply:
improve docstrings

Reply:
new issue for docstring improvements #276